kubernetes/structures_pod.go

```diff
@@ -38,7 +38,7 @@ func flattenOS(in v1.PodOS) []interface{} {
 	return []interface{}{att}
 }
 
-func flattenPodSpec(in v1.PodSpec) ([]interface{}, error) {
+func flattenPodSpec(in v1.PodSpec, isTeamplate bool) ([]interface{}, error) {
```

Comment:
Thanks for the changes @arybolovlev, should this be `isTeamplate bool` or `isTemplate bool` ?